scrapi/harvesters/biomed.py

```diff
@@ -0,0 +1,136 @@
+"""
+BioMed Central harvester of public projects for the SHARE Notification Service
+Note: At the moment, this harvester only harvests basic data on each article, and does
+not make a seperate request for additional metadata for each record.
+
+Example API query: http://www.biomedcentral.com/webapi/1.0/latest_articles.json
+"""
+
+from __future__ import unicode_literals
+
+import json
+import logging
+from dateutil.parser import parse
+
+from nameparser import HumanName
+
+from scrapi import requests
+from scrapi.base import BaseHarvester
+from scrapi.linter.document import RawDocument, NormalizedDocument
+
+logger = logging.getLogger(__name__)
+
+
+class biomedHarvester(BaseHarvester):
```

Comment:
Always start classes with an uppercase letter even if the word itself would not be capitalized.